tests/fsharp/core/members/basics/test.fs

```diff
@@ -17,6 +17,15 @@ let test (s : string) b =
 
 let check s b1 b2 = test s (b1 = b2)
 
+//--------------------------------------------------------------
+// Test defining a type using wildcard self identifiers
+
+type ClassType0 () =
+    member _.myNumber () = 0
+    member private _.myPrivateNumber () = 1
```

Comment:
As per my (late) [comment](https://github.com/dotnet/fsharp/pull/7631#issuecomment-535657384), can you instead add this test to the [tests/fsharp/core/members/self-identifier/version47/test.fs](https://github.com/dotnet/fsharp/blob/master/tests/fsharp/core/members/self-identifier/version47/test.fs) file?

Reply:
Sure, no worries.